cmd/kubeadm/app/cmd/cmd.go

```diff
@@ -17,16 +17,34 @@ limitations under the License.
 package cmd
 
 import (
+	"html/template"
```

Comment:
why are we using `html/template` when there's no html involved? might be totally fine, just want to understand. possibly deserves a comment.